src/js/pages/services/DeploymentsTab.js

```diff
@@ -1,11 +1,73 @@
+import moment from 'moment';
 import React from 'react';
+import {Table} from 'reactjs-components';
 
 import AlertPanel from '../../components/AlertPanel';
+import EventTypes from '../../constants/EventTypes';
+import MarathonStore from '../../stores/MarathonStore';
+import ResourceTableUtil from '../../utils/ResourceTableUtil';
+import StringUtil from '../../utils/StringUtil'
+
+const columnClassName = ResourceTableUtil.getClassName;
```

Comment:
Yeah I wondered about this. It's less a cache and more a set of statics. 

I was using `ServicesTable` as a reference implementation and this is one of the deviations I made. It seemed rather odd to me to recreate the whole object, complete with methods, on every render. 

I agree that from the user's perspective there is zero difference either way.